sushi-config.yaml

```
@@ -21,17 +21,17 @@ pages:
   index.md: 
     title: Introduction
   2_Specification.md:
-    title: FHIRcast specification
+    title: FHIRcast Specification
     2-1-SessionDiscovery.md:
       title: Session Discovery
     2-2-FhircastScopes.md:
-      title: OAuth2 authorization scopes
+      title: FHIRcast Scopes
```

```suggestion
      title: OAuth 2.0 Authorization Scopes
```
Don't understand the rationale for the change to FHIRcast Scopes